lib/battle/CBattleInfoCallback.cpp

```diff
@@ -707,12 +708,12 @@ TDmgRange CBattleInfoCallback::calculateDmgRange(const BattleAttackInfo & info)
 	//bonus from attack/defense skills
 	if(attackDefenceDifference < 0) //decreasing dmg
 	{
-		const double dec = std::min(0.025 * (-attackDefenceDifference), 0.7);
+		const double dec = !VLC->modh->settings.NO_ATT_AND_DEF_DAMAGE_INFLUENCE_LIMIT ? std::min(0.025 * (-attackDefenceDifference), 0.7) : 0.025 * (-attackDefenceDifference);

```

Comment:
It is better to make `0.7` and `4.0` constants configurable

Reply:
How that should be implemented for json configuration? 2 separate hardcoded features with int parameter?

Reply:
3 hardcoded features with float parameter (0.025 might be configurable too)